src/fromager/constraints.py

```diff
@@ -51,15 +51,17 @@ def load(filename: pathlib.Path | None) -> Constraints:
 # This is a helper function to find and write duplicates to start of the file
 # from constraints.txt Input: list of objects (self._build_stack in this case)
 # Returns: list of objects (processed_constraints in this case)
-def _organize_constraints(info: dict):
+def _organize_constraints(info: list[typing.Any]) -> list[typing.Any]:
```

Comment:
` list[typing.Any]` isn't a very useful type annotation. Is it a ` list[dict[str, typing.Any]]`?

Reply:
Yes that could describe it, but I was hoping to exact the exact type down later and use type alias for this cause this function accepts the `build_stack` which is a common data object across modules